common/app/googleAuth/OAuthLoginController.scala

```diff
@@ -1,19 +1,24 @@
 package googleAuth
 
 import com.gu.googleauth.{GoogleAuth, GoogleAuthConfig, UserIdentity}
-import common.{ExecutionContexts, Logging}
+import common.{Crypto, ExecutionContexts, Logging}
 import conf.Configuration
 import org.joda.time.DateTime
-import play.api.libs.Crypto
+import play.api.libs.crypto.CryptoConfig
 import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
 import play.api.mvc._
+
 import scala.concurrent.Future
 
 trait OAuthLoginController extends Controller with ExecutionContexts with implicits.Requests {
-  import play.api.Play.current
 
+  def wsClient: WSClient
```

Comment:
Lets make this implicit and remove the "implicit val ws = wsClient" in loginAction and overrideRedirectUrl